src/tools/miri/tests/fail/dyn-upcast-trait-mismatch.rs

```diff
@@ -1,5 +1,4 @@
-#![feature(trait_upcasting)]
-#![allow(incomplete_features)]
+
 
 trait Foo: PartialEq<i32> + std::fmt::Debug + Send + Sync {
```

Comment:
```suggestion
trait Foo: PartialEq<i32> + std::fmt::Debug + Send + Sync {
```

Those newlines are unnecessary :P

Reply:
I was lazy and didn't wanted to bless the test, since compiling Miri takes quite a bit of time :smile:

*but interesting, apparently Miri fail tests don't include line numbers: [dyn-upcast-trait-mismatch.stderr](https://github.com/rust-lang/rust/blob/10b9a2b81467951a7702161c2909b342c6882aa7/src/tools/miri/tests/fail/dyn-upcast-trait-mismatch.stderr). Will remove it.*